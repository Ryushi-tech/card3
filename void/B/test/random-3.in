5952398887941495
