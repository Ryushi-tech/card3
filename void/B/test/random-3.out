5952398778832387
